sdks/python/apache_beam/ml/inference/base_test.py

```diff
@@ -251,6 +281,19 @@ def test_run_inference_keyed_examples_with_unkeyed_model_handler(self):
           | 'RunKeyed' >> base.RunInference(model_handler))
       pipeline.run()
 
+  def test_drop_example_prediction_result(self):
+    def assert_drop_example(prediction_result):
+      assert prediction_result.example is None
+
+    pipeline = TestPipeline()
+    examples = [1, 3, 5]
+    model_handler = FakeModelHandlerWithPredictionResult()
```

Comment:
Shouldn't this have `drop_example=True`?

Reply:
We are passing `drop_example` via `base.RunInference`, which passes it to the `ModelHandler`.

More explanation: https://github.com/apache/beam/pull/23266#issuecomment-1248854707

Reply:
Thanks! I missed seeing the `drop=True` in the test example  😄